scaffolds/streamlit_on_cloudrun/image_classification/train_and_deploy.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>This training take a awhile if a GPU is not attached. Maybe add a line above this cell that it is normal for the training to take more than 15 mins with any accelerator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Added at the top along with the description about optional GPU. Thanks!</p>